integration_tests/mockApis/spaceBooking.ts

```diff
@@ -93,4 +93,27 @@ export default {
         status: 200,
       },
     }),
+
+  stubSpaceBookingNonArrival: placement =>
+    stubFor({
+      request: {
+        method: 'POST',
+        urlPattern: paths.premises.placements.nonArrival({
+          premisesId: placement.premises.id,
+          placementId: placement.id,
+        }),
+      },
+      response: {
+        status: 200,
+      },
+    }),
+
+  getApiPost: async (path: string): Promise<Record<string, unknown>> => {
+    const result = await getMatchingRequests({
+      method: 'POST',
+      url: path,
+    })
+    const { requests } = result.body
+    return JSON.parse(requests[0].body)
+  },
```

Comment:
That's great, may I suggest moving this to `integration_tests/mockApis/utils.ts`? Seems generic enough. Also maybe keep the `verify...` naming, so `verifyApiPost`? Would align with existing implementations.